it/utils/ETMPWiremock.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2023 HM Revenue & Customs
```

Comment:
This is actually the Regime ETMPWiremock, I deleted the non-regime one and renamed the regime one. So github consolidated the changes by making it look like RegimeETMPWiremock  was deleted and ETMPWiremock has changes. Same for compliance wiremock